core/src/blocktree_processor.rs

```diff
@@ -73,7 +77,7 @@ fn par_execute_entries(bank: &Bank, entries: &[(&Entry, LockedAccountsResults)])
 /// 2. Process the locked group in parallel
 /// 3. Register the `Tick` if it's available
 /// 4. Update the leader scheduler, goto 1
-pub fn process_entries(bank: &Bank, entries: &[Entry]) -> Result<()> {
+pub fn process_entries(bank: &Bank, entries: &[Entry], randomise_tx_execution_order: bool) -> Result<()> {
```

Comment:
Sry, we're generally following American english in the code base: s/randomise/randomize/

Reply:
Done